### ISSUE TITLE
Does automated accessibility scanning only finds ~35% of issues? Can it do better?

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#19]
As Doug reminded me, manual scans don't reveal all the accessibility issues.

Most sites need to be manually reviewed in order to meet all WCAG policies.

**Comment by bbertucc** [comment]
I do hope Equalify can advocate for better scans and WCAG rules, so 100% of A11Y issues are found.

I would also be happy to build any solution that further improves WCAG checks.

If all else fails, Equalify might need to include some sort of manual fallback so folks can create and run their own tests.

**Comment by bbertucc** [comment]
Seems like there’s no way a robot can detect contrast issues, the proper alt text on images, proper tab order, a good focus indicator, etc. Those are the manual checks - perhaps?

**Comment by bbertucc** [comment]
Happy to re-open this if anyone sees better scans as a priority.
